README.md

```diff
@@ -21,7 +21,14 @@ installation as a subdirectory of
 
 ## Restore packages
 
-	`mono src/.nuget/NuGet.exe restore src/Roslyn.sln`
+	mono src/.nuget/NuGet.exe restore src/Roslyn.sln
+
+On a fresh Mono install [that will fail with a certificate error](http://www.mono-project.com/docs/faq/security/).
```

Comment:
Thanks, I updated the wording in 8bc343d to reflect that it only applies when building Mono from source.